source/includes/api/_authentication.md

```diff
@@ -1,46 +1,54 @@
+**Warning**: treat a token in the same way you would treat a password - don't share it and always store it in a safe place (like a password manager, for example). A token is sufficient to authenticate you on the RW API, meaning that, if someone else gains access to your token, they'll be able to do anything that you would be able to do on the RW API, even without access to your Facebook/Google/Twitter login or your email and password.
 
-## How to create a new user
+## How to use your private token to authenticate a request
 
-To create a new user make a request like the one in the sidebar:
+> Example of how a request with a token is structured
 
 ```shell
-curl -X POST https://api.resourcewatch.org/auth/user \
--H "Authorization: Bearer <your-token>" \
+curl -X POST https://api.resourcewatch.org/v1/dataset \
+-H "Authorization: Bearer <your token>" \
 -H "Content-Type: application/json"  -d \
- '{
-    "email":"<email>",
-    "role":"<role>",
-    "extraUserData": {
-      "apps": [
-        "<apps>"
-      ]
-    }
+'{
+  "dataset": {
+    "name":"World Database on Protected Areas -- Global",
+    "connectorType":"rest",
+    "provider":"cartodb",
+    "connectorUrl":"https://wri-01.carto.com/tables/wdpa_protected_areas/table",
+    "application":[
+      "gfw", 
+      "forest-atlas"
+    ]
+  }
 }'
 ```
 
-There are three allowed roles: `USER`, `MANAGER` and `ADMIN`. The 'apps' field only permits applications that are powered by the API: `rw`, `gfw`, `prep`, etc.
+Once you have your token, you can use it to authenticate your requests to the RW API. Depending on which endpoints you are trying to use, certain actions may become available by simply authenticating your request. Other will require your account to have a special role or applications, which we'll cover in more detail in the [User Management](#user-management) documentation.
+
+Tokens must be provided in HTTP request header, named as `Authorization`, with the structure `Bearer: <your token>`.
+
+Once generated, a token is valid until any of the associated user information (name, email, or associated applications) changes. If your token becomes invalid, you will need to log in and go to [https://api.resourcewatch.org/auth/generate-token](https://api.resourcewatch.org/auth/generate-token) to generate a new token.
+
+
+## More information on authentication and user management
+
+This section's goal is to help you learn the very basics of how you can quickly create a user account and use your token to authenticate requests. For more in-depth details on authentication, token and user management, refer to the [User Management](#user-management) section of the documentation.
```

Comment:
IMO this paragraph should be placed at the start of the Authentication section, no need to have an extra section just for this.

Reply:
good point, will change it